src/popup/components/tool.jsx

```diff
@@ -10,13 +11,27 @@ import NoTickets from './no-tickets';
 import TicketControls from './ticket-controls';
 
 function Tool({ tickets, errors }) {
+  const { close, openopts } = useContext(EnvContext);
+
+  async function onClickSettingsLink(event) {
```

Comment:
Just outta curiosity: is this a React / JS naming pattern, to use `on<action-name><content>` for these functions? I'd personally have gone with something that indicates what the function does, rather than were it's placed: `openSettingsPage` for instance.

Reply:
Since this function is specifically a click handler calling `event.preventDefault()`, using this slightly more verbose and specific name seemed appropriate to me. I'm not sure whether there is an "official" convention here, but I've definitely seen similar naming before.